lib/components/boot-warning/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style';
+
+const BootWarning = () => {
+  return (
+    <h1>
+      Simplenote cannot currently be opened simultaneously in more than one tab
+      or window per browser.
+    </h1>
+  );
+};
```

Comment:
@SylvesterWilmott would you be willing to review this? We have this ongoing problem caused by the fact that if the app is open in more than two tabs/sessions/windows _of the same browser_ (multiple copies in Firefox /or/ multiple copies in Chrome /or/ multiple copies in Safari etc…) then the workers that store updates fight and cause the _infinite duplication_ bug.

Just need to communicate that.

Reply:
Perhaps using `<p>` instead or even `<h3>`

Aside from that, the copy looks good, I'm not sure we need the word "currently", although I understand what we're trying to communicate it does make the sentence a bit more of a mouthful.

"Simplenote cannot be opened simultaneously in more than one tab or window per browser"